scripts/ci/pipeline/sdks-archive.groovy

```diff
@@ -80,7 +80,7 @@ def archive (product, configuration, platform, chrootname = "", chrootadditional
 
                 // remove old stuff
                 sh 'git reset --hard HEAD'
-                sh 'git submodule foreach --recursive git reset --hard HEAD'
+                sh 'git submodule foreach --recursive `pwd`/scripts/ci/git-reset-hard.sh HEAD'
```

Comment:
Maybe add a note here that this should be reverted once the bug is fixed?

Reply:
well it's not gonna stop working once the bug is fixed.

and it's maybe enough of a WTF that the curious person would read the comment in `git-reset-hard.sh` if they decide to clean this up in the future.